### DISCUSSION TITLE
Using ADRs still a good idea?

### DISCUSSION THREAD

**Discussion by mleegwt** [structurizr/java#236] [General]
The Architecture Decision Records that can be referenced in the DSL are using https://github.com/npryce/adr-tools
However, that tool seems to be unmaintained since 2018 (plenty of issues and PRs present though).

Any thoughts?

**Comment by simonbrowndotje** [comment]
You can plug-in your own ADR importer (https://docs.structurizr.com/dsl/adrs), plus support for MADR + Log4brains will be built-in to the next version of the DSL (https://github.com/structurizr/java/blob/master/changelog.md). So in summary, there are many options that don't rely on `adr-tools`.
